application/factory.py

```diff
@@ -245,9 +245,9 @@ def render_error(error):
         error_code = getattr(error, "code", 500)
 
         if re.match(r"/cms", request.path):
-            return render_template("error/{0}.html".format(error_code)), error_code
+            return render_template("error/{0}.html".format(error_code)), error_code or 500
```

Comment:
I'm not sure this will work as we'll still be trying to render `error/None.html`?

I think what we want to do is change line 245 to `error_code = getattr(error, "code") or 500`, which will then provide the correct default value for both references on line 248.